client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -117,7 +118,7 @@ const ProcessingStep: Step = function ( props ) {
 	const isJetpackPowered = isNewsletterOrLinkInBioFlow( flowName );
 
 	// Currently we have the Domains and Plans only for link in bio
-	if ( flowName === LINK_IN_BIO_FLOW ) {
+	if ( flowName === LINK_IN_BIO_FLOW || FREE_FLOW ) {
```

Comment:
I imagine this is intended to be something like:
```suggestion
	if ( flowName === LINK_IN_BIO_FLOW || flowName === FREE_FLOW ) {
```
?

Also noting throughout here I see a mix of `flowName === FREE_FLOW` and `isFreeFlow( flowName )`. Is there a reason we cant be consistent with one?

Reply:
Good catch @Addison-Stavlo. This has been resolved. Now using isFreeFlow( flowName ) for the conditional.

Reply:
@Addison-Stavlo - You've also got a good point on the `flowName === FREE_FLOW` vs `isFreeFlow()` conditionals. In order to minimize changes on a big PR so we can get this out, I'd probably prefer to leave those as is. But I'll make a follow up for myself to go back through and update them all to isFreeFlow(). If we have that method, we should go ahead and use it. 

When I do that, I may clean up similar inconsistencies for `flowName === LINK_IN_BIO_FLOW` vs `isLinkInBioFlow()`. We're doing the same thing mixing those in the code.